pulser/simulation.py

```diff
@@ -74,6 +75,16 @@ def __init__(self, sequence, sampling_rate=1.0):
         self._build_basis_and_op_matrices()
         self._construct_hamiltonian()
 
+    def draw(self):
+        """Draws the simulation, showing both the input sequence and
+        the effective, constant-by-part (sampled) sequence used in QuTip.
+        """
+
+        # The effective time steps
+        time_slices = (1000*self._times).astype(int)
+
+        draw_sequence(self, time_slices)
```

Comment:
I think what you want here is 
```suggestion
        draw_sequence(self._seq, time_slices)
```
The testing error comes from there.

Reply:
The change is already made, the error comes from the test. I get 
`index 5062 is out of bounds for axis 0 with size 5011` that I'm still trying to understand

Reply:
I'll make a full review. In the meantime, use `pytest` locally to make sure all tests pass (or fix what's wrong) before pushing the changes.

Reply:
yes, I figured it's less spammy this way